src/team/invite.ts

```diff
@@ -94,24 +101,21 @@ export default async function invite(
           `You've been invited to \`[${team.tag}] ${team.name}\` by <@${msg.author.id}>! Head over to the tournaments server and use \`!team join ${team.tag}\`!\n
         If you're already in a team, you'll need to use \`!team leave\` first. You can see other teams you've been invited to with \`!invites\`.`
         );
-      const noDmMsg = `<@${u.id}> was invited, but we couldn't DM them. Tell them you've invited them!\n(<@${u.id}>, use \`!invites\` to see all your invites!)`;
-      const successMsg = `${u.username} was successfully invited! They recieved a DM from this bot.`;
-
-      if (!u.dmChannel) {
-        const ch = await u.createDM();
-        try {
-          await ch.send(embed);
-          await msg.reply(successMsg);
-        } catch {
-          await msg.reply(noDmMsg);
-        }
-      } else {
-        try {
-          await u.dmChannel.send(embed);
-          await msg.reply(successMsg);
-        } catch {
-          await msg.reply(noDmMsg);
-        }
+      const noDmMsg = warnEmbed()
+        .setTitle("Invite Successful")
+        .setDescription(
+          `<@${u.id}> was invited, but we couldn't DM them. Tell them you've invited them!\n(<@${u.id}>, use \`!invites\` to see all your invites!)`
+        );
+      const successMsg = infoEmbed()
+        .setTitle("Invite Successful")
+        .setDescription(
+          `<@${u.id}> was successfully invited! They received a DM from this bot.`
+        );
+      try {
+        await u.send(embed);
+        await msg.channel.send(successMsg);
+      } catch {
+        await msg.channel.send(noDmMsg);
```

Comment:
since these embeds are only used once lets not make them variables

Reply:
done